graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/SchemaGenerator.kt

```diff
@@ -67,7 +67,7 @@ open class SchemaGenerator(internal val config: SchemaGeneratorConfig) {
         builder.codeRegistry(codeRegistry.build())
         val schema = config.hooks.willBuildSchema(builder).build()
 
-        classScanner.close()
```

Comment:
This will be closed automatically when the schema generator goes away

Reply:
i'd still explicitly close it just in case

Reply:
since `ScanResult` implements `Closeable` we could use Kotlin `use` (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html)